Fix links

CJMinecraft01 opened the pull request:
The links did not go to the correct place (they gave a 404 error) so I have found the correct links 😄

CJMinecraft01 committed:
Fix links

The links did not go to the correct place (they gave a 404 error) so I have found the correct links 😄

tterrag1098 commented on docs/gettingstarted/autoupdate.md:
```diff
@@ -39,7 +39,7 @@ This is fairly self-explanatory, but some notes:
 * Manually inputting data can be chore. You can configure your `build.gradle` to automatically update this file when building a release, as Groovy has native JSON parsing support. Doing this is left as an exercise to the reader.
 
-Two concrete examples can be seen here for [Charset](https://gist.githubusercontent.com/Meow-J/fe740e287c2881d3bf2341a62a7ce770/raw/bf829cdefc84344d86d1922e2667778112b845b1/update.json) and [Botania Unofficial](https://gist.githubusercontent.com/Meow-J/1299068c775c2b174632534a18b65fb8/raw/42c578cf2303aa76d8900f5fdc6366122549d2a8/update.json).
+Two concrete examples can be seen here for [Charset](https://asie.pl/files/minecraft/update/charset.json) and [Botania Unofficial](https://raw.githubusercontent.com/williewillus/Botania/HEAD/version/update.json).
```
I wouldn't use a `HEAD` ref here, as that could change. Please link to a specific commit so that the URL does not break again...

CJMinecraft01 committed:
Changed HEAD to master

Could also use the non raw link which is here: https://github.com/williewillus/Botania/blob/master/version/update.json

CJMinecraft01 committed:
Changed HEAD to master

Could also use the non raw link which is here: https://github.com/williewillus/Botania/blob/master/version/update.json

CJMinecraft01 reviewed (commented):
I've changed the branch from HEAD to master. Also for a non raw link you could use this: https://github.com/williewillus/Botania/blob/master/version/update.json

tterrag1098 commented:
Sorry, master is not sufficient. You should use a specific commit.

CJMinecraft01 committed:
Put a specific commit link

CJMinecraft01 commented:
Should have changed it now to this: https://raw.githubusercontent.com/williewillus/Botania/8e86d126e10e862c032242074211b297aede087b/version/update.json sorry about that :D
